### PR TITLE
CI: update docker image

### PR DESCRIPTION
**Pull request by tjhei**
- new docker image with updated libraries
- include all information to make the docker image
- correct .pdf compilation using latexmk
- report all warnings report by biber

### CODE REVIEW

**Comment by tjhei** [comment]
FYI @marcfehling

**Comment by marcfehling** [comment]
That's great! Did you add `bibtool` to the image as well?

File: docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,39 @@
+FROM ubuntu:22.04
+
+MAINTAINER timo.heister@gmail.com
+
+USER root
+
+
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get -yq install \
+    graphviz python3 flex bison make git cmake g++ wget \
+    texlive biber bibtool \
```
Comment (tjhei) [inline docker/Dockerfile:10]: yes

Comment (marcfehling) [reply docker/Dockerfile:10]: Oh I see now in the `Dockerfile` that you did! Awesome!

**Review by marcfehling — APPROVED** [review]

**Comment by marcfehling** [comment]
The remaining 'pending' check actually passed if you click on details. Is this just a hickup of github? I'll trigger a rebuild once more to double-check.

**Comment by tjhei** [comment]
It's a leftover because I renamed the CI job while playing with it.

**Pull request merged**